cmd/record.go

```diff
@@ -303,6 +308,8 @@ func (r *Record) GetCmd() *cobra.Command {
 
 	recordCmd.Flags().StringP("command", "c", "", "Command to start the user application")
 
+	recordCmd.Flags().DurationP("keployRecorderStopTimer", "t", 0, "Command to initiate a timer that automatically stops the Keploy recorder after a specified duration of time")
```

Comment:
Make this also into something briefer like "timer to stop keploy record after the required time".